docker-compose-arm64.yml

```diff
@@ -1320,11 +1320,13 @@ services:
       VAULT_ADDR: http://edgex-vault:8200
       VAULT_CONFIG_DIR: /vault/config
       VAULT_LOCAL_CONFIG: |
-        disable_mlock = true
+        backend "file" {
+            path = "/vault/file"
+        } disable_mlock = true
```

Comment:
I agree, but this seems to be how the file is generated. In the add-security.yml file it's on a new line, but when I run make gen the compose file is constructed like this. I don't know what can be done about it. As far as I can tell mlock is still disabled when vault runs. Let me know if it seems to behave otherwise

Reply:
I agree, but this seems to be how the file is generated. In the add-security.yml file it's on a new line, but when I run make gen the compose file is constructed like this. I don't know what can be done about it. As far as I can tell mlock is still disabled when vault runs. Let me know if it seems to behave otherwise

Reply:
I agree, but this seems to be how the file is generated. In the add-security.yml file it's on a new line, but when I run make gen the compose file is constructed like this. I don't know what can be done about it. As far as I can tell mlock is still disabled when vault runs. Let me know if it seems to behave otherwise

Reply:
According to the doc https://yaml.org/spec/1.2.2/#23-scalars , use `|` instead of `>` to resolve this.
> using a literal style (indicated by “|”) where all line breaks are significant.